### PR TITLE
Feature/clcache info

### PR DESCRIPTION
**Pull request by Luschgy**

### CODE REVIEW

File: helper.psm1 [hunk 1]
```diff
@@ -189,15 +202,13 @@ Function showConfig
     Write-Host "Workdir        : "$WORKDIR
     Write-Host "Workspace      : "$env:WORKSPACE
     Write-Host "------------------------------------------------------------------------------"
+    Write-Host "Clcache Statistics"
+    configureCache
```
Comment (fceller) [inline helper.psm1]: showConfig should not call configureCache, it should only show things

File: helper.psm1 [hunk 2]
```diff
@@ -152,6 +152,19 @@ Function unlockDirectory
 # Configure Oskar
 ################################################################################
 
+Function configureCache
+{
+    If($env:CLCACHE_CL)
+    {
+	    proc -process "$(Split-Path $env:CLCACHE_CL)\cl.exe" -argument "-M 107374182400" -logfile $false -priority "Normal"
```
Comment (KVS85) [inline helper.psm1]: Just installed clcache-4.1.0 via nuget and see that binary name is `clcache.exe`. Is that correct?

File: helper.psm1 [hunk 3]
```diff
@@ -152,6 +152,19 @@ Function unlockDirectory
...
+	    proc -process "$(Split-Path $env:CLCACHE_CL)\cl.exe" -argument "-s" -logfile $false -priority "Normal"
```
Comment (KVS85) [inline helper.psm1]: Just installed clcache-4.1.0 via nuget and see that binary name is `clcache.exe`. Is that correct?

File: helper.psm1 (hunk 2 above)
Comment (Luschgy) [reply helper.psm1]: I install clcache 4.2 via zip file and rename it to work properly

File: helper.psm1 (hunk 3 above)
Comment (Luschgy) [reply helper.psm1]: I install clcache 4.2 via zip file and rename it to work properly

File: helper.psm1 (hunk 1 above)
Comment (Luschgy) [reply helper.psm1]: added new commit to fix issue

**Pull request merged**